openy.install

```diff
@@ -1275,3 +1275,21 @@ function openy_update_8095() {
     pathauto_update_8108();
   };
 }
+
+/**
+ * Uninstall Open Y AddThis and guide the user to using AddToAny as a replacement.
```

Comment:
You won't be able to uninstall addthis, because after composer update module won't be on a disk
Procedure
- create hook_update_N to uninstall module, while keeping it in composer
- release a distro release
- add this release to important versions in the documentation
- release a new version of release with module removed from composer.json ( could be done way later )

@froboy

Reply:
@podarok how quickly I forget. :) Fixed.